components/esp-tls/Kconfig

```diff
@@ -40,6 +40,13 @@ menu "ESP-TLS"
             Enable support for creating server side SSL/TLS session, available for mbedTLS
             as well as wolfSSL TLS library.
 
+    config ESP_TLS_SERVER_SESSION_TICKET_TIMEOUT
```

Comment:
Should I add another option to explicitly enable server side session ticket support instead of using CONFIG_MBEDTLS_SERVER_SSL_SESSION_TICKETS?